collectors/sentinelone/health_checks.js

```diff
@@ -0,0 +1,59 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2020, Alert Logic, Inc
+ * @doc
+ *
+ * sentinelone health checks.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+
+'use strict';
+
+const health = require('@alertlogic/al-aws-collector-js').Health;
+const RestServiceClient = require('@alertlogic/al-collector-js').RestServiceClient;
+const moment = require('moment');
+
+/* 
+* No explicit call to validate an API token, the token expires after 180 days, this 180 day timeout is refreshed 
+* upon any API calls. This call will refresh token, without being time consuming/destructive
+*/
+function sentinelOneTokenHealthCheck(callback) {
+    const hostname = process.env.paws_endpoint.replace(/^https:\/\/|\/$/g, '');
+    const token = this.secret;
+
+    let restServiceClient = new RestServiceClient(hostname);
+    restServiceClient.post('/web/api/v2.0/users/api-token-details', {
+        json: {
+            "data": {
+                "apiToken": token
+            }
+        }
+    }).then((response) => {
+        const expiresAt = moment(response.data.expiresAt).subtract(1, 'days');
```

Comment:
Can we make this a bit earlier? The token is 6 months. Could we refresh the token three weeks before it is due to expire?